src/protocols/secure_channel/CASESession.cpp

```diff
@@ -782,8 +799,8 @@ CHIP_ERROR CASESession::SendSigmaR3()
     }
 
     // Generate a signature
-    VerifyOrExit(fabric->GetEphemeralKey() != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
-    err = fabric->GetEphemeralKey()->ECDSA_sign_msg(msg_R3_Signed.Get(), msg_r3_signed_len, tbsData3Signature);
+    VerifyOrExit(mFabricInfo->GetEphemeralKey() != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
+    err = mFabricInfo->GetEphemeralKey()->ECDSA_sign_msg(msg_R3_Signed.Get(), msg_r3_signed_len, tbsData3Signature);
```

Comment:
By the time CASE is done, it's possibly not ephemeral anymore. `GetEphemeralKey()` is the wrong name. It should be `GetOperationalKey()`, which, depending on commissioning state, may be ephemeral or permanent.

Reply:
I'll update it. It used to be called `GetOperationalKey()`, but in a recent PR review got renamed to `EphemeralKey`. It makes sense that this should be called OperationalKey.